### PR TITLE
Change daemonset to use release revision not time for Helm 3 support

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/roadmap/issues/30

### CODE REVIEW

File: helm/cert-exporter/templates/daemonset.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
       labels:
         app: cert-exporter
       annotations:
-        releasetime: {{ $.Release.Time }}
+        releaseRevision: {{ $.Release.Revision | quote }}
```
Comment (tomahawk28) [inline helm/cert-exporter/templates/daemonset.yaml]: Does `$.Release.Revision` support helm v2? if not, can we check $.Release.Time is nil or not first? 
I know we are going to use it for helm3, but so many times we are deploying it without knowing which helm version for it.

Comment (tomahawk28) [reply helm/cert-exporter/templates/daemonset.yaml]: Okay I checked it, it supports 
https://v2.helm.sh/docs/chart_template_guide/#built-in-objects

Comment (rossf7) [reply helm/cert-exporter/templates/daemonset.yaml]: @tomahawk28 Thanks for checking.

Yes they actually removed Release.Revision from Helm 3 but then added it again. https://github.com/helm/helm/issues/6690

We just want to roll pods so revision also works.

**Comment by rossf7** [comment]
Tested on `gauss`.

**Pull request merged**